sdk/network/Microsoft.Azure.Management.Network/src/Properties/AssemblyInfo.cs

```diff
@@ -8,7 +8,7 @@
 [assembly: AssemblyDescription("Provides Microsoft Azure Network management functions for managing the Microsoft Azure Network service.")]
 
 [assembly: AssemblyVersion("20.0.3.0")]
```

Comment:
This need to be `20.6.1.0`, see https://stackoverflow.com/questions/64602/what-are-differences-between-assemblyversion-assemblyfileversion-and-assemblyin

Reply:
bumped the version, thanks!